docs/release-notes/release-notes-0.15.0.md

```diff
@@ -0,0 +1,10 @@
+# Release Notes
+
+## Misc
+
+* An example systemd service file for running lnd alongside a bitcoind service
```

Comment:
To satisfy the CI check, you need to include a link to the PR. You can either link the whole sentence or maybe just the `example systemd service file`.

Reply:
Oops, thank you. Done.

As a side note it seems a bit self-referential in that you need the PR number to pass the check, but you don't know the number until you open the PR (with a failing release notes check, presumably).

Reply:
Yes, that's true. Though for the release notes it is nice to link to the PR that added the change so you can easily look up all the context if you're interested. So unless you're guessing the next PR number from looking at the last created issue/PR you need to push at least a second time, unfortunately.